src/components/ui/command.tsx

```diff
@@ -0,0 +1,134 @@
+'use client'
+
+import * as React from 'react'
```

Comment:
Remove barrel import of React and import forwardRef only. Have a look at the other components